pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -186,7 +186,7 @@ func (pas *PodAutoscalerStatus) MarkScaleTargetInitialized() {
 	podCondSet.Manage(pas).MarkTrue(PodAutoscalerConditionScaleTargetInitialized)
 }
 
-// IsSKSReady returns true if the PA condition denoting that SKS is ready.
+// IsSKSReady returns true if the PodAutoscaler's SKS is ready.
```

Comment:
Let's remove this entire function if it ain't needed anymore? I ignored it on the other PR but since we're fixing nits anyway :man_shrugging:

Reply:
heh, yeah, sgtm, one sec

Reply:
🪓